build.sh

```diff
@@ -2,7 +2,5 @@
 
 cd src
 pdflatex main.tex
-pdflatex main.tex
```

Comment:
could you reinstate these, if I remember correctly they are needed when we start to run bibtex for the references.